OpenAI_API/Chat/ChatMessage.cs

```diff
@@ -48,16 +49,22 @@ public ChatMessageRole Role
 			}
 		}
 
-		/// <summary>
-		/// The content of the message
-		/// </summary>
-		[JsonProperty("content")]
-		public string Content { get; set; }
+        /// <summary>
+        /// The content of the message
+        /// </summary>
+        [JsonProperty("content", NullValueHandling = NullValueHandling.Include)]
+        public string Content { get; set; }
 
 		/// <summary>
 		/// An optional name of the user in a multi-user chat 
 		/// </summary>
 		[JsonProperty("name")]
 		public string Name { get; set; }
-	}
+        /// <summary>
+        /// Optional field function call
+        /// The name and arguments of a function that should be called, as generated by the model.
+        /// </summary>
+        [JsonProperty("function_call")]
+        public Function_Call Function_Call { get; set; }
```

Comment:
We can follow standard naming convention here and still use the json name as-is - more changes ensue down the line...
```suggestion
        public FunctionCall FunctionCall { get; set; }
```